src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4130,23 +4130,34 @@
      *
      * <p>For example:</p>
      * <pre>
-     *         String hello = """
-     *                 Hello, world!
-     *                 """;
+     * public class MyClass {
+     *     String example = """
+     *             Hello,
+     *             World!
+     *             """;
+     * }
      * </pre>
      * <p>parses as:</p>
      * <pre>
-     * |--VARIABLE_DEF
-     * |   |--MODIFIERS
-     * |   |--TYPE
-     * |   |   `--IDENT (String)
-     * |   |--IDENT (hello)
-     * |   |--ASSIGN (=)
-     * |   |   `--EXPR
-     * |   |       `--TEXT_BLOCK_LITERAL_BEGIN (""")
-     * |   |           |--TEXT_BLOCK_CONTENT (\n                Hello, world!\n                    )
-     * |   |           `--TEXT_BLOCK_LITERAL_END (""")
-     * |   `--SEMI (;)
+     * CLASS_DEF
+     * |--MODIFIERS
+     * |   `--LITERAL_PUBLIC (public)
+     * |--LITERAL_CLASS (class)
+     * |--IDENT (MyClass)
+     * `--OBJBLOCK
+     *     |--LCURLY ({)
+     *     |--VARIABLE_DEF
```

Comment:
should be `VARIABLE_DEF -&gt; VARIABLE_DEF` please correct all subtree

Reply:
done